ScrumPoker.Data/Data/GameRoomDataBase.cs

```diff
@@ -0,0 +1,47 @@
+using ScrumPoker.Core.Models;
+using ScrumPoker.DataBase.Interfaces;
+
+namespace ScrumPoker.Data.Data;
+
+public class GameRoomDataBase : IGameRoomData
+{
+    private static List<GameRoom> _gameRooms = new List<GameRoom>();
+    private static int Id { get; set; }
+    
+    public void Create(string name)
+    {
+        var gameRoom = new GameRoom
+        {
+            Name = name,
+            Id = ++Id
+        };
+        
+        _gameRooms.Add(gameRoom);
+    }
+
+    public List<GameRoom> GetAllGameRooms()
+    {
+        return _gameRooms;
+    }
+
+    public void DeleteAllRooms()
+    {
+        _gameRooms.Clear();
+    }
+
+    public void DeleteRoomByName(string name)
```

Comment:
These should be `DeleteById` and `GetById`
Later on multiple players will be able to create multiple rooms with the same name

Reply:
So change the param + name + all the services and controllers where this is used